### PR TITLE
Code to prepare ERA5-Land forcing for AMS tutorial 2024

### PR DESCRIPTION
**Pull request by CharlesZheZhang**
Modify the codes to generate ERA5-Land forcing and initial conditions for AMS tutorial 2024.

### CODE REVIEW

File: hrldas/HRLDAS_forcing/run/examples/ERA5/Prepare_ERA5-Land_Forcing.sh [hunk 1]
```diff
@@ -19,11 +19,18 @@
...
+inmonth=11
+inday=01
+preyear=2023
+premonth=10
+preday=31
 
-let last_year=$year-1
-PREVIOUS_YEAR=/fas4/anav/INPUTS/ERA5_for_Noah-MP/YEARLY/ERA5-Land_Noah-MP_$last_year.grb
+indate=${inyear}-${inmonth}-${inday}
+INFILE=../../../ERA5_forcing/ERA5-Land-Noah-MP-${indate}.grib
```
Comment (cenlinhe) [inline hrldas/HRLDAS_forcing/run/examples/ERA5/Prepare_ERA5-Land_Forcing.sh:31]: this needs to be INFILE=../../../ERA5_forcing/ERA5-Land-Noah-MP-${inyear}\_${inmonth}\_${inday}.grib, or you can rename your ERA5 input data file name. Similarly for the PREVIOUS_FILE below

**Comment by cenlinhe** [comment]
Please also submit a PR to develop branch to update the default ERA5 script in HRLDAS_forcing folder

File: hrldas/HRLDAS_forcing/run/examples/ERA5/extract_era5_init.py [hunk 2]
```diff
@@ -10,24 +10,27 @@
...
 
 vars_name = ["SKT","SD","STL1","STL2","STL3","STL4","SWVL1","SWVL2","SWVL3","SWVL4"]
 vars_time = ["hr fcst:","hr fcst:","hr fcst:","hr fcst:","hr fcst:","hr acc:","hr acc:","hr acc:"]
 
-data_dir = "/glade/scratch/zhezhang/ERA5/land/"
-infile  = data_dir+"ERA5_global_2001-10-01_init.grib"
-results_dir = "/glade/scratch/zhezhang/ERA5/land/extract/INIT/"
+data_dir = "../../../ERA5_forcing/"
+infile  = data_dir+"ERA5-Land-Noah-MP-2023_11_01_INIT.grib"
+results_dir = "./OUTPUT/INIT/"
```
Comment (cenlinhe) [inline hrldas/HRLDAS_forcing/run/examples/ERA5/extract_era5_init.py:22]: This should be "./OUTPUTS/INIT/"

**Review by tslin2 — COMMENTED** [review]
will be good to revise the line 3-13 in [Prepare_ERA5-Land_Forcing.sh] as well

**Comment by cenlinhe** [comment]
I am going to close this issue since we do not need the "AMS tutorial" branch anymore.

**Pull request closed without merging**